src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -46,17 +46,15 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
         promptSteps.push(subscriptionStep);
     }
 
-    promptSteps.push(...[
-        new vscAzureAppService.SiteNameStep(),
-        new ResourceGroupListStep(),
-        new vscAzureAppService.AppServicePlanListStep()
-    ]);
-    LocationListStep.addStep(wizardContext, promptSteps);
+    promptSteps.push(new vscAzureAppService.SiteNameStep());
```

Comment:
Not sure I understand the purpose of this change... is CustomLocationListStep **not supposed** to see the `AppServicePlanListStep`?

Reply:
CustomLocationListStep automatically creates a plan when a custom location is selected, so the AppServicePlanListStep will be skipped.

Reply:
got it

Reply:
What I was missing is that the `CustomLocationListStep.addStep()` is a static method that creates and adds an instance of `CustomLocationListStep` to `promptSteps`. For some reason this line looked to me like a method on a `CustomLocationListStep` instance, which was taking a list of "prompt steps" 🤪 Now I understand.